src/migration_utilities.js

```diff
@@ -9,4 +9,30 @@ const argv = require("yargs")
   .version(false)
   .argv
 
-module.exports = argv
\ No newline at end of file
+function getDependency(artifacts, network, deployer, path) {
+  let Contract
+
+  if (network === "development" || network === "coverage") {
+    Contract = artifacts.require(path)
+  } else {
+    const contract = require("truffle-contract")
+
+    Contract = contract(require(path))
+    Contract.setProvider(deployer.provider)
+    Contract.setNetwork(network2id[network.replace("-fork", "")])
+  }
+  return Contract
+}
+
+const network2id = {
```

Comment:
This naming is pretty poor. If it is a constant it should be capitalized. Also, isn't there a way to get this from the truffle config object?